.github/rdmo-app-ci-script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+git clone https://$GITHUB_TOKEN@github.com/rdmorganiser/rdmo.git
+# need rdmo only for testing/config/settings
+git clone https://$GITHUB_TOKEN@github.com/rdmorganiser/rdmo-app.git
```

Comment:
I think the pluging can be tested in `rdmo/testing/` alone, right? So cloning rdmo (no token needed) woulsd suffice.